src/main/java/io/phasetwo/service/resource/Converters.java

```diff
@@ -2,104 +2,214 @@
+                        });
+        i.setAttributes(attr);
+        return i;
+    }
+
+    public static Invitation convertInvitationModelToInvitation(InvitationModel e) {
+        Invitation i =
+                new Invitation()
+                        .id(e.getId())
+                        .email(e.getEmail())
+                        .createdAt(e.getCreatedAt())
+                        .inviterId(e.getInviter().getId())
+                        .invitationUrl(e.getUrl())
+                        .organizationId(e.getOrganization().getId())
+                        .roles(Lists.newArrayList(e.getRoles()));
+        i.setAttributes(Maps.newHashMap(e.getAttributes()));
+        return i;
+    }
+
+    public static LinkIdp convertIdpLinkModelToIdpLInk(IdentityProviderModel e, RealmModel realm) {
+        var idp = new LinkIdp();
+        idp.setAlias(e.getAlias());
+        String firstBrokerLoginFlowId = e.getPostBrokerLoginFlowId();
+        if (firstBrokerLoginFlowId != null) {
+            AuthenticationFlowModel flow = realm.getAuthenticationFlowById(firstBrokerLoginFlowId);
+            if (flow == null) {
+                throw new ModelException("Couldn't find authentication flow with id " + firstBrokerLoginFlowId);
+            }
+            idp.setPostBrokerFlow(flow.getAlias());
+        }
+
+        Map<String, String> config = e.getConfig();
+        if (config != null) {
+            idp.setSyncMode(config.get(IdentityProviderModel.SYNC_MODE));
+        }
+        return idp;
+    }
+
+    public static OrganizationRepresentation convertOrganizationModelToOrganizationRepresentation(OrganizationModel organizationModel,
+                                                                                                  RealmModel realm,
+                                                                                                  ExportOptions options) {
+        var organization = convertOrganizationModelToOrganizationAttributes(organizationModel);
+        var roles = organizationModel
+                .getRolesStream()
+                .map(Converters::convertOrganizationRoleRepresentation)
+                .toList();
+        var idpOptional = organizationModel
+                .getIdentityProvidersStream()
+                .filter(identityProviderModel -> idpInOrg(identityProviderModel, organizationModel.getId()))
+                .map(idp -> convertIdpLinkModelToIdpLInk(idp, realm))
+                .findFirst();
+
+        var organizationRepresentation = new OrganizationRepresentation();
+        organizationRepresentation.setOrganization(organization);
+        organizationRepresentation.setRoles(roles);
+        idpOptional.ifPresent(organizationRepresentation::setIdpLink);
+
+        if (options.isUsersIncluded()) {
+            var members = organizationModel.getMembersStream()
+                    .map(userModel -> {
+                        var userRoles = organizationModel.getRolesByUserStream(userModel)
```

Comment:
Use the new `getRolesStreamByUser` method to get the roles.

Reply:
Done